tests/integration/utils.py

```diff
@@ -122,21 +122,21 @@ def _calculate_files_checksums_in_dir(root_dir: str) -> Dict:
     files_checksums = {}
 
     for dir_, _, files in os.walk(root_dir):
+        rel_dir = PurePath(dir_).relative_to(root_dir)
         for file_name in files:
-            rel_dir = os.path.relpath(dir_, root_dir)
-            rel_file = os.path.join(rel_dir, file_name)
+            rel_file = rel_dir.joinpath(file_name)
```

Comment:
Btw, the division operator also works as joinpath
```python
rel_file = rel_dir / file_name
```
No need to change this everywhere, just a fun tip :slightly_smiling_face: